Assignment6.Rmd

```diff
@@ -66,4 +67,6 @@ table(M2$certified, M2$predict1)
 
 table(M2$certified, M2$predict2)
 
+printcp(c.tree2)
 ```
```

Comment:
This is a good interpretation, you might want to re-frame your analysis with respect to training and test sets though. What can you say about the generalizability of your model given its performance on the new data set?